hack/builder/common.sh

```diff
@@ -1,5 +1,6 @@
 DOCKER_PREFIX=${DOCKER_PREFIX:-"quay.io/kubevirt"}
 DOCKER_IMAGE=${DOCKER_IMAGE:-"builder"}
+CROSS_COMPILE_IMAGE=${CROSS_COMPILE_IMAGE:-"builder-cross-compile"}
```

Comment:
`DOCKER_CROSS_IMAGE` perhaps? To maintain the naming convention.

Related: I think that `-compile` is redundant, and it should be just called `builder-cross`.

Reply:
Yes, i'm not sure about the naming scheme here. I can change the variable easily enough. For the image i'm also considering just adding `-cross` or `-cross-compile` to the tag and not publish it under a completely new name, since it technically isn't really. I'm open to suggestions what the name of the image should be.

Reply:
It's not a clear-cut decision, but personally I think it makes more sense for `-cross` to appear in the name of the image than in that of the tag. The rationale being, that different tags usually mean "roughly same components, different versions" while different images usually mean "significantly different components", and the cross image fits better into the latter bucket IMO.

Reply:
I have renamed the variable to `DOCKER_CROSS_IMAGE`.

For shortening the image name, i have no problem with that. @brianmcarey If you agree, could i bother you to rename it on quay.io, than i can change it in the code as well.

Reply:
Ok so it is going to become - `http://quay.io/kubevirt/builder-cross:2406170946-2735000ca` ?

Reply:
I think that is the plan, yes.

Reply:
That  quay.io repo is in place now.